$OpenBSD: patch-subvertpy_wc_c,v 1.1 2016/04/26 15:10:58 jasper Exp $

Be sure to include the proper header file to get correct symbol visibility
information.

--- subvertpy/wc.c.orig	Tue Sep 27 12:27:30 2011
+++ subvertpy/wc.c	Sat Apr 16 11:55:44 2016
@@ -27,6 +27,7 @@
 
 #include "util.h"
 #include "editor.h"
+#include "wc.h"
 
 #ifndef T_BOOL
 #define T_BOOL T_BYTE
